### PR TITLE
Updated Amazon S3 connector and fixed index

### PR DESCRIPTION
**Pull request by dhirsch1**
Updated Amazon S3 connector and fixed index. Would like to merge these changes into the beta branch. 

Note: Most of the links from the index.adoc topic are not working for any connector, at least in my local build.

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ In the `Create Object` operation, set the `Content Length` to a value greater th
 
 == To Use an AWS KMS Master Key
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc:11]: Remove "To" from headings.

File: modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc [hunk 2]
```diff
@@ -33,3 +33,12 @@ doc:id="DOC_ID" >
 
 [NOTE]
 The access key and secret key are not listed in the connector configuration. Instead, the correct values are used from the values specified in the settings.
+
+== Next Step
+
+You can try out the xref:amazon/amazon-s3-connector-examples.adoc[Example].
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc]: Please change to Kim-type wording, such as: "After you understand how to configure a master key and provide credentials, you are ready to try the [Examples]."

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 3]
```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
...
 
-. Configure the target component for the connector.
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
 
-== Configure the Amazon S3 Connector in Design Center
+For information about Amazon S3 fields, see the xref:amazon/amazon_S3_connector-reference[Amazon S3 Connector Reference].
 
-To configure the Amazon S3 connector in Design Center:
+== Configure the Trigger
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: Please use "Configure the Input Source Trigger" -- we can't assume everyone gets the trigger concept.

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 4]
```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
...
-. Click the plus sign.
-. Select a trigger. To have an Amazon S3 event initiate access to your app (that is, to trigger your app), select the Amazon S3 Connector and then select one of the Amazon S3 trigger operations. Alternatively, select an HTTP Listener or Scheduler as the trigger.
+To configure a trigger:
 
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the Trigger card.
+. If you are using your connector as an input source trigger, search for the connector and select an operation. Otherwise, search for and select *HTTP* or *Scheduler*.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: "your connector" is incorrect. The connector is not something a user owns, but merely uses. How about "this connector" ?

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 5]
```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
...
+. Select an operation for the connector.
+. Enter the required values in the *General* tab.
+. If you are:
++
+** Using the default Amazon S3 storage, leave the default entries for the *Proxy* and the *Advanced* tabs.
+** Connecting to a different storage than the default AWS S3, specify its URL in the the *Advanced* tab's *S3 Compatible Storage URL* field.
++
+. If needed, enter values for other tabs.
+. Specify access information to the connector resource, as described below.
+. If a *Test* button is available, click it to test the connection.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: This generic instruction is not good -- it just sounds like the writer didn't review the UI. The Amazon S3 connector does have a Test button.

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc [hunk 6]
```diff
@@ -4,42 +4,58 @@ include::_attributes.adoc[]
...
 +
-* For the *Basic* connection type:
+Validating a connection with Test Connection requires that you have permission in AWS IAM to the action s3:ListAllMyBuckets. If you don’t have this permission, the test fails. However, you can still use the connector and the operations to which you have access.
```
Comment (george-hoffer) [inline modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc:50]: There's no "Test Connection" button in Design Center. The button name is "Test".

File: modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc (hunk 3 above)
Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: Is this the title we are using going forward?

Comment (dhirsch1) [reply modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc]: I will add this now. I think it needs to be discussed:

Configure the Input Source (Trigger)
Configure the Input Source Trigger

**Pull request merged**